### PR TITLE
Reactive ListView with StreamController

### PR DESCRIPTION
**Pull request by yurisasc**
Simulated how to implement a reactive ListView by using StreamController.
Uses simple planet facts as local data that is shown to the UI periodically.

Link to Dartpad: https://dartpad.dev/d0dcc4ce4ec83e63a9816d4a6e0f3fed

### CODE REVIEW

**Comment by mkiisoft** [comment]
https://dartpad.dev/b6409e10de32b280b8938aa75364fa7b is the default gist for increment counter, please, provide your own gist/dartpad.

File: README.md [hunk 1]
```diff
@@ -134,6 +134,10 @@ If you appreciate the content 📖, support projects visibility, give 👍| ⭐|
 
   [<img src = "/art/buttons.gif" width=240>](https://dartpad.dev/214eee8a820ba4bcff73279a93281a53)
 
+- [Reactive-ListView](https://dartpad.dev/b6409e10de32b280b8938aa75364fa7b) by [Yuris Cakranegara](https://https://github.com/yurisasc)(04/01/2020)
```
Comment (mkiisoft) [inline README.md]: provide your own gist/dartpad and not default one

File: README.md [hunk 2]
```diff
@@ -134,6 +134,10 @@ If you appreciate the content 📖, support projects visibility, give 👍| ⭐|
...
+
+  [<img src = "/art/ReactiveListView.gif" width="240">](https://dartpad.dev/b6409e10de32b280b8938aa75364fa7b)
```
Comment (mkiisoft) [inline README.md]: same for this one

**Review by mkiisoft — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (yurisasc) [reply README.md]: Thank you, and sorry for the trouble.

**Review by divyanshub024 — APPROVED** [review]

**Comment by divyanshub024** [comment]
@mkiisoft Thanks a lot for reviewing the PR.

**Pull request merged**